Makefile

```diff
@@ -18,7 +19,7 @@ all:
 	$(MAKE) -C po
 
 po-pull:
-	rpm -q --whatprovides zanata-python-client &>/dev/null || ( echo "need to run: dnf install zanata-python-client"; exit 1 )
+	which $(ZANATA_CLIENT_BIN) &>/dev/null || ( echo "need to install zanata python client package"; exit 1 )
 	zanata pull $(ZANATA_PULL_ARGS)
```

Comment:
If we're going to use a variable, it should be used everywhere. I'd also name it $ZANATA so it isn't so long.

Reply:
So true. Fixed.